### PR TITLE
[CLOUD-390] update filebeat to 7.17.9

### PR DESCRIPTION
**Pull request by gibchikafa**

### CODE REVIEW

**Comment by SirOibaf** [comment]
@gibchikafa - what's the purpose of this PR. The changes should have been done in the docker-repo not here. Can we close this one?

**Comment by gibchikafa** [comment]
Filebeat service also doesn't start without adding this configuration.

**Comment by SirOibaf** [comment]
On which version?

**Comment by gibchikafa** [comment]
on ubuntu22.04. It's either we update filebeat or add that configuration to the current filebeat version.

File: templates/default/filebeat.yml.erb [hunk 1]
```diff
@@ -29,3 +29,10 @@ logging.files:
     permissions: 0644
 
 path.data: <%= node['filebeat']['base_dir'] %>/data/<%= @log_name %>
+
+seccomp:
```
Comment (SirOibaf) [inline templates/default/filebeat.yml.erb:33]: We need to update filebeat.

**Review by SirOibaf — CHANGES_REQUESTED** [review]

**Comment by gibchikafa** [comment]
Yes, I updated yesterday but the binaries were not on master yet on repo.hops.works. I reverted to current version to proceed with the test.

File: attributes/default.rb [hunk 2]
```diff
@@ -67,8 +67,8 @@
 #
 # Filebeat
 
-default['filebeat']['version']                  = "7.13.2"
-default['filebeat']['url']                      = "#{node['download_url']}/opensearch/filebeat-oss-#{node['filebeat']['version']}-linux-x86_64.tar.gz"
+default['filebeat']['version']                  = "7.17.2"
+default['filebeat']['url']                      = "#{node['download_url']}/opensearch/filebeat-#{node['filebeat']['version']}-linux-x86_64.tar.gz"
```
Comment (SirOibaf) [inline attributes/default.rb:71]: We need to use the oss version.

**Review by SirOibaf — CHANGES_REQUESTED** [review]

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**